Source/Fuse.Controls.Primitives/Image.Visual.uno

```diff
@@ -83,16 +99,39 @@ namespace Fuse.Controls
 
 			if (Container.StretchMode == StretchMode.Scale9)
 			{
-				Fuse.Elements.Internal.Scale9Rectangle.Impl.Draw(dc, this, ActualSize, GetSize(), tex, color, 
-					Scale9Margin);
+				Fuse.Elements.Internal.Scale9Rectangle.Impl.Draw(dc, this, ActualSize, GetSize(), tex, color, Scale9Margin);
 			}
 			else
 			{
+
+
+				var flip = Matrix.Scaling(1,1,1);
+				var rotation = Matrix.RotationZ(0.0f);
+
+				if (!IgnoreExif)
+				{
+					if (Source.Orientation.HasFlag(ImageOrientation.FlipVertical))
+						flip = Matrix.Scaling(1,-1,1);
+
+					if ((Source.Orientation & (int)0x03) == ImageOrientation.Rotate270)
+						rotation = Matrix.RotationZ(Math.PIf / 2.0f);
+					else if ((Source.Orientation & (int)0x03) == ImageOrientation.Rotate90)
+						rotation = Matrix.RotationZ(-Math.PIf / 2.0f);
+					else if ((Source.Orientation & (int)0x03) == ImageOrientation.Rotate180)
+						rotation = Matrix.RotationZ(Math.PIf);
+
+				}
+
+				var translateToCenter = Matrix.Translation(0.5f,0.5f,0.0f);
+				var translateBack = Matrix.Translation(-0.5f,-0.5f,0.0f);
+				var imageTransform = Matrix.Mul(translateBack, flip, rotation, translateToCenter);
```

Comment:
I think we should make a helper-method to compute the image-transform from the orientation...